packages/components/src/Banner/__snapshots__/Banner.spec.tsx.snap

```diff
@@ -53,7 +53,7 @@ exports[`<Banner /> AlertDismissable story renders snapshot 1`] = `
   class="bannerDialog horizontal dismissable colorAlert"
 >
   <button
-    class="button dismiss colorAlert button variantLink colorAlert"
```

Comment:
The `CloseButton` is using the "link" variant because we hadn't even added the "plain" variant yet, hence the disparity.